resources/AddTeam.html

```diff
@@ -1,9 +1,36 @@
-<h4>Add a team</h4>
-
-<form action="/addteam">
-    <label for="number">Team number:</label> <input type="text" id="number" name="number">
-    <br />
-    <label for="matches">Matches (comma-separated):</label> <input type="text" id="matches" name="matches">
-    <br />
-    <input type="submit" value="Add team">
-</form>
+<!DOCTYPE html>
+
+<html>
+
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Scouting App | Add Team</title>
+    <link rel="stylesheet" href="css/AddTeam.css">
+  </head>
+
+  <body>
+    <div class="content-container">
+      <div class="content-container-position">
+        <h4 id="addteam-title">Add a team</h4>
+
+        <form action="/addteam">
+
+          <div class="label-input-div">
+            <label for="number">Team number:</label> <input type="text" id="number" name="number" placeholder="Team number">
+          </div>
+
+          <div class="label-input-div">
+            <label for="matches">Matches (comma-separated):</label> <input type="text" id="matches" name="matches" placeholder="Matches">
+          </div>
+
+          <input type="submit" value="Add team">
+
+        </form>
+      </div>
+    </div>
+
+    <script> </script> <!-- This is line is here to prevent the pageload animation bug in chrome -->
```

Comment:
```suggestion
    <!-- This line is here to prevent the pageload animation bug in Chrome -->
    <!-- See https://bugs.chromium.org/p/chromium/issues/detail?id=332189#c20 -->
    <script> </script> 
```